Doc/library/__main__.rst

```diff
@@ -361,8 +358,5 @@ defined in the REPL becomes part of the ``__main__`` scope::
     >>> namely.print_user_name()
     Jabberwocky
 
-Note that in this case the ``__main__`` scope doesn't contain a ``__file__``
-attribute as it's interactive.
-
```

Comment:
We may want to keep this line, depending on the resolution of the issue. I agree with the changes above.